test/unit/concerns/symbolize_test.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+class SymbolizeTest < ActiveSupport::TestCase
+  class UsageLimit < ApplicationRecord
+    self.table_name = 'usage_limits'
+    include Symbolize
+    symbolize :period
+
+    # Because it is null: false in DB
+    before_save :set_plan_type
+
+    private
+    def set_plan_type
+      self.plan_type ||= 'Plan'
+    end
+  end
+
+  def setup
+    Timecop.freeze
+  end
+
+  def teardown
+    Timecop.return
```

Comment:
Ok, good :+1: I see that the teardown is executed even when there is an error in a test.

![image](https://user-images.githubusercontent.com/11318903/74238056-3dfda700-4cd5-11ea-8730-af0630be9fc6.png)

![image](https://user-images.githubusercontent.com/11318903/74238100-58d01b80-4cd5-11ea-821c-28d9342720d0.png)